.github/workflows/build-deploy-dev.yml

```diff
@@ -5,42 +5,133 @@ on:
     types: [opened, reopened, synchronize, closed]
     branches:
       - main
+  workflow_dispatch:
+    inputs:
+      bumb:
+        description: "The type of semantic version increment to make. By default it is 'prerelease'."
+        required: false
+        default: "prerelease"
+        type: choice
+        options:
+          - major
+          - premajor
+          - minor
+          - preminor
+          - patch
+          - prepatch
+          - prerelease
+      preId:
+        description: "Identifier to be used to prefix version increments. By default it is 'prerelease' for pull request event and workflow dispatch will follow the last release tag prefix, if any."
+        required: false
+        type: choice
+        options:
+          -
+          - premajor
+          - preminor
+          - prepatch
+          - prerelease
+      releaseType:
+        description: "Select the release type. By default only tag will be created. if you want to create a release with the tag select the type."
+        type: choice
+        required: false
+        options:
+          -
+          - draft
+          - publish
+
 
 jobs:
   # Print variables for logging and debugging purposes
   checkEnv:
     name: Check Env variables
     environment: DEV
     runs-on: ubuntu-latest
-    if: ${{ github.event.pull_request.merged == true}}
+    if: github.event.pull_request.merged == true || github.event_name == 'workflow_dispatch'
     steps:
       - name: Print Env Vars
         run: |
           echo Git Base Ref: ${{ github.base_ref }}
-          echo Git Build ID: ${{ github.event.number }}
           echo Git Pull Request Ref: ${{ github.event.pull_request.head.sha }}
           echo OC CLI Version: $(oc version)
 
+  # Create new tag.
+  createTag:
+    name: "Create tag"
+    if:  github.event.pull_request.merged == true || github.event_name == 'workflow_dispatch' 
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get bumb value
```

Comment:
For my understanding: what is a bumb value? 😄

Reply:
😄 actually `bumb` is the variable that determines the type of semantic version increment to make.  major, premajor, minor, preminor, patch, prepatch, or prerelease are the choices, for example, if the last tag was `v1.0.0`, and then we assigned `bumb === `major`` then it will create `v2.0.0` as the next tag. I used the sample variable that was used in the action https://github.com/zwaldowski/semver-release-action#bump, if you have any meaningful name in mind, I don't mind changing it